payments/gateways/base.py

```
@@ -0,0 +1,100 @@
+
+        Returns:
+            Dict containing:
+                - reference: Unique payment reference/ID from the gateway
+                - url: Checkout URL for the user
+                - amount: Total amount (Decimal)
+                - raw_data: Full response from gateway (dict)
+
+        Raises:
+            ValueError: If user not found or invalid parameters
+            Exception: If gateway API call fails
+        """  # noqa: E501
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)